test/gtest/common/xpmem_test.h

```diff
@@ -0,0 +1,100 @@
+/**
+ * Copyright (c) NVIDIA CORPORATION & AFFILIATES, 2023. ALL RIGHTS RESERVED.
+ *
+ * See file LICENSE for terms.
+ */
```

Comment:
space line after (in other files as well)

Reply:
fixed once